src/components/SearchBox/SearchBox.scss

```diff
@@ -0,0 +1,50 @@
+.searchbox {
+    $input-font-size: 30px;
+    $input-line-height: 70px;
+    $font-family: Roboto Slab, sans-serif;
+
+    position: relative;
+    width: 500px;
+    margin: 50px auto;
+
+    input {
+        height: 60px;
+        width: 100%;
+        min-width: 100%;
+        padding: 0;
+        border-radius: 0;
+        line-height: $input-line-height;
+        background-color: transparent;
+        color: #757575;
```

Comment:
Lets make this color a variable with the other variables and store in a different vars file?

Reply:
Not yet since the color is not standardized yet. This is a first pass at getting all components coded up. Color scheme, design updates followed up with Fit and Finish will take place later.

Reply:
Either way you should move it to a variable file and change the color there. This will allow fewer color changes in different files when you standardize a color.